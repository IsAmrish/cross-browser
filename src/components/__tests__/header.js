import React from "react";
import renderer from "react-test-renderer";
import Header from "../header";

describe('Header component test', () => {
  it('Render Header', () => {
    const tree = renderer.create(<Header siteTitle="Cross Browser header" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});