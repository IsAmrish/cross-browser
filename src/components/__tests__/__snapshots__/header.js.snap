// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test Render Header 1`] = `
<header
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      style={
        Object {
          "marginBottom": "8px",
          "textAlign": "center",
        }
      }
    >
      <a
        aria-current="page"
        className=""
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        Cross Browser header
      </a>
    </h1>
    <p
      style={
        Object {
          "color": "white",
          "lineHeight": "30px",
          "margin": "0px",
          "textAlign": "center",
        }
      }
    >
      This project is to list out those css properties whose usage way is different in different browsers
    </p>
  </div>
</header>
`;
